mscore/exportaudio.cpp

```diff
@@ -254,7 +257,12 @@ bool MuseScore::saveAudio(Score* score, const QString& name)
                   if ((mode & QIODevice::WriteOnly) == 0) {
                         return false;
                         }
-                  sf     = sf_open(qPrintable(filename), SFM_WRITE, &info);
+
+                  file.setFileName(filename);
+                  if (!file.open(mode)) {
+                      return false;
```

Comment:
old code style, 6 spaces and banner style indent ;-)

Reply:
fixed :)